packages/web-app-files/src/helpers/space/sortMembers.ts

```diff
@@ -0,0 +1,17 @@
+import { spaceRoleManager } from '../share'
+
+export const sortSpaceMembers = (shares: Array<any>): Array<any> => {
+  const sortedManagers = shares
+    .filter((share) => share.role.name === spaceRoleManager.name)
+    .sort((a, b) => {
+      return a.collaborator.displayName.localeCompare(b.collaborator.displayName)
+    })
+
+  const sortedRest = shares
```

Comment:
Is Rest the correct word ?

Reply:
I think so, why not? We divide in managers and the rest of the roles.